openapi.yaml

```diff
@@ -525,343 +507,331 @@ parameters:
       - RH
       - EF
       - MP
+    description: |
+      Filter by phone type. Common phone types and their descriptions:
+      - `CM` - Current
+      - `PA` - Student Alternate Contact
+      - `RK` - Financial Aid Address
+      - `MP` - Mobile Phone
+responses:
+  '400':
+    description: Bad request
+    schema:
+      $ref: '#/definitions/ErrorResult'
+  '404':
+    description: Resource not found
+    schema:
+      $ref: '#/definitions/ErrorResult'
+  '500':
+    description: Internal server error
+    schema:
+      $ref: '#/definitions/ErrorResult'
+securityDefinitions:
+  OAuth2:
+    type: oauth2
+    flow: application
+    tokenUrl: https://api.oregonstate.edu/oauth2/token
+    scopes:
+      full: Full access to the API
 definitions:
```

Comment:
I think it'll be better to pause my review here for a while, otherwise my comments will be too many.